data/json/items/comestibles/brewing.json

```diff
@@ -1,4 +1,24 @@
 [
+  {
+    "type": "COMESTIBLE",
+    "id": "brew_mycus_wine",
+    "name": "mycus wine must",
+    "description": "Unfermented mycus wine.  A goopy white mess, made from berries of marloss.  It smells sweetly of mushrooms.",
```

Comment:
This says the wine must is made of marloss berries, but the recipe uses mycus fruits.